<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lbLanguage">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbLanguage"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbDateFormat">
           <property name="text">
            <string>Date format</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <widget class="QLineEdit" name="leDateFormat"/>
         </item>
         <item row="2" column="0" rowspan="2">
          <widget class="QLabel" name="lbTimeFormat">
           <property name="text">
            <string>Time format</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="leTimeFormat"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUseSystemDateTimeFormat">
         <property name="text">
          <string>Use system date/time format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbFullScreenMode">
         <property name="text">
          <string>Show main window on full screen at startup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbShowSumsWithCurrency">
         <property name="text">
          <string>Show money sums in same column with currency</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbShowTableGrid">
         <property name="text">
          <string>Show grid in tables</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbShowLineNumbers">
         <property name="text">
          <string>Show line numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbResizeTableRowsToContents">
         <property name="text">
          <string>Resize table rows to contents</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUseTableAlternateColors">
         <property name="text">
          <string>Use alternate row colors in table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUseSystemFontsAndColors">
         <property name="text">
          <string>Use system fonts and colors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTableFont">
         <property name="text">
          <string>Table font</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbColors">
         <property name="title">
          <string>Colors</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnGridColor1">
            <property name="text">
             <string>Grid base</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGridColor2">
            <property name="text">
             <string>Grid alternate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColumns">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
